package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "@hyperlane-xyz/registry",
   "description": "A collection of configs, artifacts, and schemas for Hyperlane",
-  "version": "6.11.0",
+  "version": "6.12.0-http.2",
   "dependencies": {
+    "express": "^4.21.2",
```

Comment:
wonder if it makes sense to spin this off to a side-package so it doesn't add to the normal registry size

Reply:
alternatively I can do something similar to what we did with the `FileSystemRegistry` for tree shaking
https://github.com/hyperlane-xyz/hyperlane-registry/pull/34